.babelrc

```diff
@@ -1,14 +1,15 @@
 {
     "env": {
-        "production": {
-            "presets": [
-                "latest"
-            ]
-        },
+        "production": {},
         "development": {
             "plugins": [
                 "transform-async-to-generator"
             ]
+        },
+        "admin": {
+            "presets": [
+                "latest"
```

Comment:
I know this is admin, but maybe it's worth using https://github.com/babel/babel-preset-env so we can specify browser versions (like we do with autoprefixer).